rundeckapp/grails-app/conf/spring/resources.groovy

```diff
@@ -757,6 +757,8 @@ beans={
         initParams = configParams?.toProperties()?.collectEntries {
             [it.key.toString(), it.value.toString()]
         }
+
+        useForwardHeaders = Boolean.getBoolean('rundeck.jetty.connector.forwarded')
```

Comment:
this is only reading the java system property, this should support the app config